bubbles/commands/help.py

```diff
@@ -13,7 +13,7 @@ def format_text(data: Dict) -> str:
 
 def help(payload):
     plugins_with_help = dict()
-    for plugin in PluginManager.plugins:
+    for plugin in payload["extras"]["meta"].plugins:
```

Comment:
I'd really like to completely change this `payload` stuff someday and have something properly typed here instead of the weird dict that just contains everything somewhere.

But this would be out of scope for this PR.

Reply:
If you're opposed to the dict, a dataclass would work well!